src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball1 = new Lottery(). getRandomBall();
+        Ball ball2 = new Lottery(). getRandomBall();
+        Ball ball3 = new Lottery(). getRandomBall();
+        System.out.println(ball1 + "\n" + ball2 + "\n" + ball3);
```

Comment:
I mean that you should create a Lottery instance outside the loop, and then inside the loop, you should create and print random balls.

Lottery lottery = new Lottery();
for(...) {
...
}